oioioi/mp/models.py

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from oioioi.base.utils.deps import check_django_app_dependencies
+
+from oioioi.participants.models import RegistrationModel
+from oioioi.contests.models import Contest
+
+check_django_app_dependencies(__name__, ['oioioi.participants'])
+
+
+class MPRegistration(RegistrationModel):
+    terms_accepted = models.BooleanField(_("terms accepted"), default=False)
+
+    def erase_data(self):
+        self.terms_accepted = False
+        self.save()
+
+
+class SubmissionScoreMultiplier(models.Model):
```

Comment:
This is a very good basic model. Out of scope for MP this model could be much more universal. Allowing for both start and end dates. Linking multipliers to rounds or even ProblemInstances would make it more versatile too.